Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <Import Project=".\scripts\sdk\Sdk.props" />
 
   <PropertyGroup>
-    <LangVersion>8</LangVersion>
+    <LangVersion>9</LangVersion>
```

Comment:
So support for C# 9.0 added
(Requires Visual Studio 16.8 which should be OK.)
Do you have a list what features that are supported with .NET 4 framework?

Reply:
This actually not required, I was tinkering with something else.
Nullable annotations came with C# 8.0.